Composer/packages/client/src/recoilModel/atoms/appState.ts

```diff
@@ -369,3 +369,33 @@ export const showGetStartedTeachingBubbleState = atom<boolean>({
   key: getFullyQualifiedKey('showGetStartedTeachingBubbleState'),
   default: false,
 });
+
```

Comment:
Consider a new module called authState for these atoms.

Reply:
done!